catalyst_voices/packages/catalyst_voices_localization/lib/l10n/intl_en.arb

```diff
@@ -708,6 +708,20 @@
   "yes": "Yes",
   "no": "No",
   "total": "Total",
+  "file": "file",
+  "key": "key",
+  "upload": "Upload",
+  "browse": "browse",
+  "uploadDropInfo": "Drop your {itemNameToUpload} here or ",
+  "uploadProgressInfo": "Upload in progress",
+  "uploadKeychainTitle": "Upload Catalyst Keychain",
+  "@uploadKeychainTitle": {
+    "description": "A title on keychain upload dialog"
+  },
+  "uploadKeychainInfo": "Upload Catalyst Keychain",
```

Comment:
It's the same as `uploadKeychainTitle`, is this correct?

Reply:
good call